### DISCUSSION TITLE
is GitHub Classroom GDPR-friendly?

### DISCUSSION THREAD

**Discussion by mekhatria** [community/community#141763] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi,
I am creating Highcharts Academy using GitHub classrooms, and I enjoy it :)

However, the HR has raised concerns about if GitHub Classroom is GDPR-friendly, they event requested a signed version.

Could you please assist me to solve this challenge?

**Comment by lin-ycv** [comment]
part of GDPR is that you can request deletion of your data, but based on this "feature":
https://www.youtube.com/watch?v=ZFtgPjsieTc
I would say it's not 100% GDPR compliant

how exactly are you usign Highcharts Academy with GitHub classrooms? maybe there's alternatives

**Reply by mekhatria** [reply]
Thx for your message.
It is just a standard GitHub classroom setup, requiring only the GitHub username :)

**Reply by mekhatria** [reply]
@lin-ycv :
A bit off-topic but still related to the main objective:
I'm currently developing Highcharts Academy, and I'd be happy if you and/or your students would be interested in testing it out through early access.
Let me know what you think, and we can also schedule a meeting to discuss it further if you'd like.

**Comment by MorelliniMatteo** [comment]
Creating an educational platform like Highcharts Academy using GitHub Classroom is a great initiative! However, addressing GDPR (General Data Protection Regulation) concerns is crucial, especially when dealing with student data. Here’s how you can approach this challenge:

Understanding GDPR in the Context of GitHub Classroom
Data Protection Principles:

Lawfulness, Fairness, and Transparency: Ensure that data collection and processing activities are lawful, fair, and transparent to users.
Purpose Limitation: Collect personal data for specified, legitimate purposes and do not process it further in a manner incompatible with those purposes.
Data Minimization: Only collect data that is necessary for your specific purpose.
Accuracy: Keep personal data accurate and up to date.
Storage Limitation: Retain personal data only for as long as necessary.
Integrity and Confidentiality: Ensure appropriate security measures to protect personal data.
User Consent:

Ensure that students and participants in your academy are informed about how their data will be used and obtain their consent. This can include a privacy policy or agreement that outlines your data practices.
Assessing GitHub Classroom for GDPR Compliance
Data Controller vs. Data Processor: Understand whether GitHub (and GitHub Classroom) acts as a data controller or a data processor in the context of your educational activities. This distinction is important for determining your responsibilities regarding data protection.

GitHub's Privacy Policy: Review GitHub's privacy policy to understand how they handle personal data. GitHub provides information on their GDPR compliance, including details about data processing and security measures.

Data Processing Agreement (DPA): GitHub offers a Data Processing Agreement for organizations that use their services. Ensure you have access to this agreement, as it outlines GitHub's commitments regarding data processing.

Steps to Address HR Concerns
Gather Documentation:

Obtain and compile documentation from GitHub that explains their GDPR compliance and data handling practices. This may include links to their GDPR FAQ, privacy policy, and DPA.
Create a Privacy Policy:

Draft a privacy policy for your Highcharts Academy that outlines how you will collect, use, and protect student data. This document should include:
The types of personal data you collect.
The purposes for data processing.
How long data will be retained.
Students' rights regarding their data (access, rectification, erasure, etc.).
Provide a Signed Document:

If HR requires a signed version, reach out to GitHub support or your GitHub organization account manager to request a signed DPA or other relevant agreements. This document can serve as evidence of GitHub's compliance.
Communicate with HR:

Prepare a brief report or presentation for HR summarizing your findings and actions. This should include:
A summary of GitHub’s GDPR compliance.
Your privacy policy for the academy.
Any signed agreements or relevant documentation.
Additional Considerations
Education and Training: Consider providing training for educators and staff involved in the academy on data protection best practices to ensure compliance.

Regular Reviews: Conduct regular reviews of your data practices and GitHub's policies to stay updated on any changes that may affect GDPR compliance.

Resources
[GitHub Privacy Policy](https://github.com/site/privacy)
[GitHub GDPR FAQ](https://github.com/blog/2705-gdpr-and-github)
[GitHub Data Protection Agreement](https://github.com/contact)
